python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -44,9 +44,9 @@ async def create(
 
     Args:
         client (TGlideClient): The client to execute the command.
-        indexName (TEncodable): The index name.
+        index_name (TEncodable): The index name.
         schema (List[Field]): Fields to populate into the index. Equivalent to `SCHEMA` block in the module API.
-        options (Optional[FtCreateOptions]): Optional arguments for the FT.CREATE command. See `FtCreateOptions`.
+        options (Optional[FtCreateOptions]): Optional arguments for the FT.CREATE command.
```

Comment:
Why do you remove the see?

Reply:
In case of python, unlike JAVA and Node, this statement does not link to the actual class and is therefore not adding any extra value.